sw/device/lib/testing/mock_mmio.h

```diff
@@ -141,6 +141,20 @@ Int ToInt(std::initializer_list<BitField> fields) {
  */
 inline internal::LittleEndianBytes LeInt(const char *bytes) { return {bytes}; }
 
+Int Inverted(const char *bytes) {
+  char *inverted_bytes = malloc(sizeof(Int));
```

Comment:
malloc is best avoided in C++. I think you meant for this to be `new char[sizeof(Int)]`? Though even then, I think a stack allocation is wisest.